pkgs/development/libraries/science/chemistry/cppe/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchFromGitHub, cmake
+, python3, enablePython ? false
+}:
+
+let
+  pythonWP = python3.withPackages (p: with p; [
```

Comment:
```suggestion
  pythonEnv = python3.withPackages (p: with p; [
```
or simply py

Reply:
Changed in favour of passing all python packages directly, unfortunately still producing conflicts, see [here](https://github.com/NixOS/nixpkgs/pull/144253#discussion_r741921129)

Reply:
Followed the approach in https://github.com/NixOS/nixpkgs/pull/144253#discussion_r742069051, which solves the problems with much less hacks.